app/src/main/java/com/kuit/kuit6android/ui/home/component/RecentlyViewedRow.kt

```diff
@@ -0,0 +1,84 @@
+package com.kuit.kuit6android.ui.home.component
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.kuit.kuit6android.R
+import com.kuit.kuit6android.ui.home.data.RestaurantData
+
+@Composable
+fun RecentlyViewedRow(modifier: Modifier = Modifier) {
+    val restaurantList = listOf(
+        RestaurantData(
+            imageId = R.drawable.img_ourddeokbokki,
+            name = "아워떡볶이",
+            time = "30분",
+            rate = 4.9f,
+            reviewNum = 3849
+        ),
+        RestaurantData(
+            imageId = R.drawable.img_konoha,
+            name = "코노하 건대점",
+            time = "25분",
+            rate = 4.9f,
+            reviewNum = 2744
+        ),
+        RestaurantData(
+            imageId = R.drawable.img_twozzim,
+            name = "두찜 건대점",
+            time = "30분",
+            rate = 4.7f,
+            reviewNum = 795
+        ),
+        RestaurantData(
+            imageId = R.drawable.img_xihongshi,
+            name = "시홍쓰",
+            time = "40분",
+            rate = 4.4f,
+            reviewNum = 1143
+        ),
+        RestaurantData(
+            imageId = R.drawable.img_opoke,
+            name = "오포케",
+            time = "15분",
+            rate = 3.9f,
+            reviewNum = 6847
+        )
+    )
+    Column (
+        modifier = modifier.padding(start = 27.dp)
+            .fillMaxWidth()
+    ){
+        Text(
+            text = "최근 본 맛집",
+            color = Color.Black,
+            fontSize = 17.sp,
+            fontWeight = FontWeight.Bold,
+            modifier = modifier.padding(bottom = 25.dp)
+        )
+        LazyRow(
+            horizontalArrangement = Arrangement.spacedBy(21.dp)
+        ) {
+            items(restaurantList) { restaurant ->
+                RecentlyViewedItem(restaurantData = restaurant)
+            }
+        }
```

Comment:
```suggestion
            items(restaurantList) { restaurant ->
                RecentlyViewedItem(restaurantData = restaurant)
            }
            item { 
                Spacer(modifier = Modifier.width(27.dp))
            }
        }
        
```

추가로 마지막에 여백 넣는건 이런식으로 하시면 됩니다!
사실 Spacer 외에도 itemsIndexed로 마지막 인덱스인지 판별해서 넣는 방법도 있지만, 위와 같은 방법이 가장 간단할 거 같습니다!